coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -33,3 +38,93 @@ def __init__(self, documentation, language, docstyle,
 
     def __str__(self):
         return self.documentation
+
+    def parse_documentation(self):
+        """
+        Parses documentation independent of language and docstyle.
+
+        :return:
+            A list of the parsed documentation metadata.
+        :raises:
+            NotImplementedError
+        """
+        if self.language == "python" and self.docstyle == "default":
+            return self.parse_python_default()
+        else:
+            raise NotImplementedError(
+                "Documentation parsing for {0.language!r} in {0.docstyle!r}"
+                " has not been implemented yet".format(self))
+
+    def parse_python_default(self):
```

Comment:
you should make this one "private", if C is specified as lang nobody should have access to that one (more or less) :)